jep/0000/README.adoc

```diff
@@ -0,0 +1,227 @@
+= JEP-0000: Readonly system configuration
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Readonly system configuration
+
+| Sponsor
+| link:https://github.com/timja[timja]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| 2020-01-20
+
+| BDFL-Delegate
+| link:https://github.com/oleg-nenashev[oleg-nenashev]
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins-ci.org/browse/JENKINS-12548[JENKINS-12548]
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+This JEP introduces a method to allow system 'configuration' to be viewed but not changed.
+
+== Specification
+
+Any methods that check for `Jenkins.ADMINISTER`, `Jenkins.RUN_SCRIPTS`, `PluginManager.CONFIGURE_UPDATECENTER`, and `PluginManager.UPLOAD_PLUGINS` will be evaluated to determine if the use of the new `Jenkins.SYSTEM_READ` permission type is appropriate. 
+
+The general rule of thumb is that all views should be readable by `Jenkins.SYSTEM_READ`, unless they give no benefit to the user.
+An example of a non beneficial view is the 'Script console' view, this is too dangerous for non administrators,
+and there is no useful information on here so the permissions on this view won't be changed.
+
+By default, all of the configuration sections in the `/manage` page require `Jenkins.ADMINISTER`, this will be changed to `Jenkins.SYSTEM_READ`. This is safe as plugins will have additional permission checks on `Jenkins.ADMINISTER` on form submission, to prevent existing CSRF attacks.
```

Comment:
What would the user experience look like in this scenario?  Do they see an error page when trying to save their changes, or does it just silently not persist their changes?

Reply:
For pages that have been updated they won't see the apply button.
If the page hasn't been updated they will get a permission denied error when they click the apply button